lib/cylc/cycling/__init__.py

```diff
@@ -32,9 +30,12 @@ def parse_exclusion(expr):
                         "permitted" % expr)
     else:
         remainder, exclusions = expr.split('!')
-        if '/' in exclusions:
-            raise Exception("'%s': exclusion must be at the end of the "
-                            "expression" % expr)
+        if ',' in exclusions:
+            if (not exclusions.strip().startswith('(') or not
+                    exclusions.strip().endswith(')')):
+                raise Exception("'%s': a list of exclusions must be "
```

Comment:
It would be better to use here custom type of Exception instead of general Exception itself.